nix/nixpkgs.nix

```diff
@@ -17,6 +17,7 @@ let
         sha256 = "1jcyd9jy7kz5zfch25s4inwlivb1y1w52fzfjy5ra5vcnp3hmqyr";
         fetchSubmodules = true;
       };
+      patches = [];
```

Comment:
We don’t need this do we?

Reply:
I've added a comment to clarify. Upstream nixpkgs adds patches that are incompatible with our version override.

Reply:
Ah I see, thanks for the explanation!